src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
```

Comment:
The value '40' is a literal and should be replaced with a constant that describes what this value represents.